molgenis-data-excel/src/main/java/org/molgenis/data/excel/simple/ExcelWriter.java

```diff
@@ -0,0 +1,86 @@
+package org.molgenis.data.excel.simple;
```

Comment:
A correct name would be OfficeOpenXmlWorkbookWriter or OoXmlWorkbookWriter (https://en.wikipedia.org/wiki/Office_Open_XML) or XlsxWriter since we don't support writing xlsm. Personally I prefer ```XlsxWriter```.